.github/workflows/ci.yml

```diff
@@ -59,10 +59,10 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: 3.9
-      - name: Install setuptools and wheel
-        run: python -m pip install --upgrade --user setuptools wheel
+      - name: Install build
+        run: python -m pip install --user build
       - name: Build sdist and wheel
-        run: python setup.py sdist bdist_wheel
+        run: python -m build
       - name: Publish distribution PyPI
         uses: pypa/gh-action-pypi-publish@master
```

Comment:
I'd recommend pinning this and then using dependabot to update it every so often (it doesn't change that often).

Reply:
Using Dependabot is a great idea, I think there are more places where we could benefit from it. As with your other suggestion, I'd prefer to handle this in a separate PR.

Reply:
Happy to make those in a followup if (since) you like the ideas.